include/z64player.h

```diff
@@ -158,7 +159,7 @@ typedef enum {
 } PlayerDoorType;
 
 
-#define PLAYER_LIMB_BUF_COUNT PLAYER_LIMB_MAX + 2 // 2 extra entries in limb buffers?
+#define PLAYER_LIMB_BUF_COUNT ALIGN8(PLAYER_LIMB_MAX)
```

Comment:
How is alignment on an array length supposed to help with alignment in memory?

Reply:
this is assuming we keep them typed as vec3s right? I didnt really follow the conversation, but there was some talks about not even typing the buffers as vec3s anymore. not sure what the merit is in doing that and what the positives and negatives are, just parroting what i read in discord (and elliptics comments above)

Reply:
I guess we go with the above suggestion and talk about the buffer thing later

Reply:
Oh yeah, sorry, I haven't got time to update this PR. Will do now

Reply:
yea no worries, I meant more so the discussion on the plain buffer vs Vec3s thing, but that should probably happen outside of this pr. so romans suggestion is fine